pom.xml

```diff
@@ -446,6 +447,7 @@
               <exclude>.classpath</exclude>
               <exclude>.project</exclude>
               <exclude>**/dependency-reduced-pom.xml</exclude>
+              <exclude>**/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister</exclude>
```

Comment:
Did we try just adding license file using # character as comment ? It should work and avoid rat exclusion